doc/tutorials/optimizer/optimizer.txt

```diff
@@ -1,494 +0,0 @@
-/*!
-
-@file optimizer.txt
```

Comment:
Can you remove the reference from https://github.com/mlpack/mlpack/blob/master/doc/tutorials/tutorials.txt?

Reply:
yup, i've removed the references